namerd/docs/config.md

```diff
@@ -47,6 +47,23 @@ Stores the dtab in memory.  Not suitable for production use.
 
 * *namespaces* -- Optional.  A map of namespaces to corresponding dtabs.
 
+### K8sDtabStore
+
+`io.buoyant.namerd.storage.experimental.k8s`
```

Comment:
For consistency with the zookeeper store description in this document, I think this should be written as:

```
### K8sDtabStore
*experimental*
  
`io.buoyant.namerd.storage.experimental.k8s`
```